clingo/plan.sh

```diff
@@ -28,7 +30,9 @@ do_build() {
     -DCMAKE_BUILD_TYPE=Release \
     -DCMAKE_INSTALL_PREFIX="${PREFIX}" \
     -DCLINGO_MANAGE_RPATH=YES \
-    -DCLINGO_BUILD_APPS=YES
+    -DCLINGO_BUILD_APPS=YES \
+    -DLUA_LIBRARIES="$(hab pkg path core/lua)/lib/liblua.a" \
```

Comment:
I forget the exact rationale, but I seem to remember @smacfarlane advising to use the `pkg_path_for` helper instead of `hab pkg path`

Possibly the former gives you some guarantee that the Lua you're referring to is the one that satisfies this package's dependency, and the latter gives the first/latest build of Lua that hab can find?

Reply:
Updated.